packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1964,6 +1964,9 @@ export default mixins(
 						node.disabled = false;
 					}
 
+					// If the user create new worklow, make sure that the default node's disabled property is reseted
+					node.disabled = false;
```

Comment:
this would affect a lot more than the default nodes, because `addNode` is called in a few different places.. 
also since you are fixing this, can you also ensure the start node position is reset to its original place

Reply:
@mutdmour I just pushed and fixed this.  Also I added fix to reset the start node position to its original place.